hubspot/provider.go

```diff
@@ -0,0 +1,10 @@
+package hubspot
```

Comment:
We are getting an over-proliferation of files inside of each connector directory, which increases the overhead of creating new connectors. Can this live in `string.go`? It might also make sense to rename that file to `strings.go`  and consolidate `name.go`, `url.go` to all live in that file.

Reply:
Moved Name() & Provider() inside `strings`, all HTTP method files into one `methods.go` file and `Close` into `client.go`